modules/installing-cluster-loader.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/using-cluster-loader.adoc
+
+[id='installing_cluster_loader_{context}']
+= Installing Cluster Loader
+Cluster Loader is included in the `atomic-openshift-tests` package.
+
+.Procedure
```

Comment:
@sjug This procedure seems incomplete. What does the user have to do once the file is edited?

Reply:
@sjug PTAL :eyeglasses:

Reply:
Once it's removed from the exclusions you can install the package `yum install atomic-openshift-tests`

Reply:
@sjug Thanks! Updated. :fireworks: